runner/fixtures/runner.pipeline.json

```diff
@@ -132,12 +132,28 @@
             "modified_date": "2019-12-05T01:12:39.854Z",
             "name": "access legacy msi",
             "github": "git@github.com:mskcc/ACCESS-Pipeline",
-            "version": "master",
+            "version": "ij/output_bam_files_instead_of_directory",
```

Comment:
I should probably rename this branch to something like "beagle_compat", as it now has more than just the single compatibility change for getting it to work with beagle